doc/whats_new/v1.0.rst

```diff
@@ -473,11 +473,6 @@ Changelog
   :func:`inspection.permutation_importance`. :pr:`19411` by :user:`Simona
   Maggio <simonamaggio>`.
 
-- |API| :class:`inspection.PartialDependenceDisplay` exposes a class method:
```

Comment:
I probably miss something but why did you remove this entry?

Reply:
was a merge conflict issue, mistakenly removed. Put it back, thanks for noticing.